CoordImporter/Windows/MainWindow.cs

```diff
@@ -15,7 +15,7 @@
 
 namespace CoordImporter.Windows;
 
-public class MainWindow : Window, IDisposable
+public sealed class MainWindow : Window, IDisposable
```

Comment:
i'm not sure why you sealed this XD but i left it that way

Reply:
I think I made it sealed because that's what Sylvanas had in the refactor that they had worked on. I never looked into why but it still compiled after putting in sealed so I left it there.


#BestSoftwareEngineeringPractices

Reply:
aha XD